src/vivarium_gates_mncnh/components/antenatal_care.py

```diff
@@ -23,30 +21,6 @@
 from vivarium_gates_mncnh.utilities import get_location
 
 
-class TreeMachine(Machine):
```

Comment:
These classes got moved to tree.py and were made more configurable for future decision trees.